extensions/amp-fx-collection/0.1/test/integration/test-amp-fx-fade-in-scroll.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+const ANIMATION_TIME = 50;
```

Comment:
Where is this default defined on the runtime code?

Reply:
It is set to zero in the template by the browser needs to render another frame. This could probably be updated to use `window.requestAnimationFrame` but that seems like a bigger change.

Reply:
Could we add a comment/TODO here?